var/spack/repos/builtin/packages/dbcsr/package.py

```diff
@@ -45,15 +46,16 @@ class Dbcsr(CMakePackage, CudaPackage, ROCmPackage):
     )
 
     variant("opencl", default=False, description="Enable OpenCL backend")
+    variant("mpi_f08", default=False, description="Use mpi F08 module")
 
     depends_on("blas")
     depends_on("lapack")
     depends_on("mpi", when="+mpi")
     depends_on("libxsmm@1.11:~header-only", when="smm=libxsmm")
 
     depends_on("cmake@3.17:", type="build", when="@2.1:")
-    depends_on("cmake@3.10:", type="build")
-    depends_on("py-fypp", type="build")
+    depends_on("cmake@3.22:", type="build")
+    depends_on("py-fypp@3.1", type="build")
```

Comment:
Should this be a minimum version instead of an exact version? What about older dbcsr versions?

```suggestion
    depends_on("py-fypp", type="build")
    depends_on("py-fypp@3.1:", type="build", when="@2.6:") # maybe 2.6:?
```

Reply:
Actually we can leave the default `depends_on("py-fypp", type="build")` as it is already updated to the latest version of the tool